<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lab_02</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Сила тока</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QChartView" name="chartI">
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>22</red>
             <green>22</green>
             <blue>22</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Сопротивление</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QChartView" name="chartR">
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>22</red>
             <green>22</green>
             <blue>22</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Напряжение на конденсаторе</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QChartView" name="chartU">
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>22</red>
             <green>22</green>
             <blue>22</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Температура</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QChartView" name="chartT">
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>22</red>
             <green>22</green>
             <blue>22</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Напряжение на лампе</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QChartView" name="chartIR">
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>22</red>
             <green>22</green>
             <blue>22</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Input values</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>I0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Lк</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineL">
         <property name="text">
          <string>0.000187</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="buttonCalculate">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>τ</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>tp</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineTau">
         <property name="text">
          <string>0.000001</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineR">
         <property name="text">
          <string>0.35</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineT">
         <property name="text">
          <string>0.001</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineU">
         <property name="text">
          <string>1400</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Uc0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineI">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Rк</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineL</tabstop>
  <tabstop>lineR</tabstop>
  <tabstop>lineU</tabstop>
  <tabstop>lineI</tabstop>
  <tabstop>lineTau</tabstop>
  <tabstop>lineT</tabstop>
  <tabstop>buttonCalculate</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>chartI</tabstop>
  <tabstop>chartR</tabstop>
  <tabstop>chartU</tabstop>
  <tabstop>chartT</tabstop>
  <tabstop>chartIR</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
